### DISCUSSION TITLE
火大！蓝灯今天更新到7.9.6版本了，依然无法正常使用。

### DISCUSSION THREAD

**Discussion by ycjdixon** [getlantern/lantern-forum-cn#977] [General]
几个服务器跳来跳去，无论跳到哪个都无法上网。蓝灯的团队要摆烂到什么时候？别忘了是收费的啊！

**Comment by MaxMKMax** [comment]
@ycjdixon 非常对不起啊大哥. 不是我们摆烂, 使网络限制和防火墙一直更新, 我们速度有点追不上. 您帮我提交个应用内报告? 然后给我票号. 明晚我会和技术团队开会. 我帮您提醒一下!

**Reply by ycjdixon** [reply]
@Mike777Khoo 那为什么我这边其他家免费的节点都能用上呢？只是免费的节点有限流和限量。

**Comment by ycjdixon** [comment]
@Mike777Khoo 你要的票号：78303131
